sql/sql_partition.cc

```diff
@@ -253,20 +253,13 @@ static bool is_name_in_list(const char *name, List<String> list_names)
     FALSE                         Success
 */
 
-bool partition_default_handling(TABLE *table, partition_info *part_info,
+bool partition_default_handling(Partition_handler *part_handler,
```

Comment:
Since you are touching the declaration anyway, might as well make it static and MY_ATTRIBUTE((warn_unused_result))